### PR TITLE
Codepipeline trigger source experiment

### PR DESCRIPTION
**Pull request by caggle**
~~As discussed, testing Github contents to an S3 bucket on each commit, and triggering a build from the changes of S3 bucket instead.~~

In this PR I will try to resolve the dilemma of creating or re-using the github webhook in the CF template (which is to be used for creating multiple stacks, per resource). 

If I cannot find a viable solution, we can start exploring Github -> S3 -> Codepipeline, as described here: https://medium.com/allermedia-techblog/cloudformation-example-set-codepipeline-to-trigger-with-s3-file-change-events-c51b746eacee

### CODE REVIEW

**Comment by caggle** [comment]
So with the last commit https://github.com/mozilla/mozdef-event-framework-template/pull/4/commits/71d7b55c257bcba618ab83577e2d256d2bc34be9 above, when I try to deploy a stack with `CreateGitHubWebHook` set to blank for this repo (which means: "don't create a webhook because there already exists one"), the stack is created no problem. However, when a push is made to this repo, the pipeline does not start. This is _not_ what we want.

This makes me think that because we are creating a webhook with CF and associating it with the codepipeline, there is a one-to-one relationship between a webhook and a stack deployed with that pipeline, even if the pipeline uses the same Github repo with an existing webhook. This explains the presence of multiple webhooks. Although I am not sure if this is something we want.

**Comment by caggle** [comment]
I found a solution to this problem in the above commit https://github.com/mozilla/mozdef-event-framework-template/pull/4/commits/39c02f7a4fa8c8c272c1206bb217ddf5755c30eb. If we remove the creation of the webhook from the CF template (mainly `PipelineWebhook` and where it's referenced) we no longer run into issues. Build also happens on pushes, without a webhook, but polling by AWS (see `PollForSourceChanges: true` on line 461). I have deployed a pipeline with this and verified it works as intended.

This was how it was defined in the CF template originally, before I made changes to the CF template in https://github.com/mozilla/mozdef-event-framework-template/pull/1 (with the exception of fetching Github personal access token from Secrets Manager.

So basically, with the last commit
- There are no webhooks,
- Github integration still uses a personal access token which is stored in the AWS secrets manager,
- Setting up the Github integration becomes a one-time manual process, which needs to be done for each github repo. Since we are considering to use a single Github repo (this), I have done this one-time setup already, so it can be re-used with multiple stacks using different resources.

@Phrozyn let me know what you think. If you are OK with this approach, I will remove the commented out parts and can merge after.

**Comment by Phrozyn** [comment]
Hi Cag, Yes I'm good with this approach :) Ty!

**Review by Phrozyn — APPROVED** [review]
Preliminary approval, won't merge this as I know you want to remove the commented sections.

**Comment by caggle** [comment]
Thanks! Removed the commented parts about webhook and merging.
